nservicebus/upgrades/5to6.md

```diff
@@ -298,7 +298,7 @@ If your are using it just include [the source code](https://github.com/Particula
 
 ## Transport seam
 
-`IDispatchMessages` have been obsoleted and is replaced by `IPushMessages`. The interfaces are equivalent so if you're implementing your own transport you should be able to just implement the new interface.
+`IDispatchMessages` have been obsoleted and is replaced by `IPushMessages`. The interfaces are equivalent so if you're implementing your own transport you should be able to just implement the new interface. It's worth to add, that `PushContext` has been given a new property `PushContext.ReceiveCancellationTokenSource`, revealing the intent of cancellation for receiving the current message. Your transport implementation should act accordingly, cancelling the receive when the source's token is cancelled.
```

Comment:
I was using the same style as used before 'you're implementing your own transport', that was the reason. Ok, will apply.